ajax/libs/nipplejs/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "nipplejs",
+  "version": "0.6.7",
+  "filename": "nipplejs.min.js",
+  "description": "A virtual joystick for touch capable interfaces",
+  "author": "Yoann Moinet <yoann.moinet@gmail.com>",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/yoannmoinet/nipplejs.git"
+  },
+  "homepage": "http://yoannmoinet.github.io/nipplejs",
+  "keywords": [
+    "virtual",
+    "joystick",
+    "touch"
+  ],
+  "npmName": "nipplejs",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    }
+  ]
```

Comment:
@ajay2507 No need to modify the file rule. Just the auto-update method. I've edited my earlier comments to indicate the same 👍